components/ClubCard.js

```diff
@@ -72,38 +73,39 @@ const CardTitle = s.strong`
   margin-bottom: 0.5rem;
 `
 
-class ClubCard extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      modal: '',
-    }
-  }
+const shorten = desc => {
```

Comment:
I don't know if it applies in this case, but you could try using [text-overflow: ellipsis](https://www.w3schools.com/cssref/css3_pr_text-overflow.asp) to show exactly the amount of text that can fit inside the box.

Reply:
Issue with the CSS solution is it only works for one line of text—there's no easy way to have like 3 lines of text where the last line ends in a '...' if needed

Reply:
Looks like [css-line-clamp](https://caniuse.com/#feat=css-line-clamp) is a thing but not supported well on all browsers. Ah well, maybe sometime in the future.